database/db_modules.js

```diff
@@ -11,6 +11,11 @@ const getAllReviews = function(callback) {
 
 const getReviewsById = function(id,callback) {
```

Comment:
You might want to rename it to `getReviewsByRestaurantId` in case someone gets confused by which ID you are talking about (I thought it might refer to the Review ID not the Restaurant ID).